File: data/src/main/java/dev/olog/data/api/lastfm/album/AlbumSearch.java

package dev.olog.data.api.lastfm.album;

import androidx.annotation.Nullable;

import com.google.gson.annotations.Expose;

public class AlbumSearch {

    @Expose
    @Nullable
    public Results results;

    public static class Results {

        @Expose
        @Nullable
        public Albummatches albummatches;
    }


}

File: data/src/main/java/dev/olog/data/api/lastfm/album/AlbumSearch.kt
package dev.olog.data.api.lastfm.album

import kotlinx.serialization.Serializable

@Serializable
class AlbumSearch(
    val results: Results? = null,
) {

    @Serializable
    class Results(
        val albummatches: Albummatches? = null,
    )
}